quickstart/overview.markdown

```diff
@@ -0,0 +1,26 @@
+---
+title: Overview of the ALICE O<sup>2</sup>
+layout: main
+---
+
+This is an overview of the O<sup>2</sup> system. It is pretty rough, but it should give you an idea of the general architecture. Please refer to other documents such as the [TDR](https://cds.cern.ch/record/2011297).
+
+## Requirements
+
+1. After LS2 the ALICE experiment will have gone through a massive upgrade to handle minimum bias PbPb interactions at 50 kHz. This will results in a 100 times higher event rate than in Run 1 (2010).
```

Comment:
```suggestion
1. After LS2 the ALICE experiment will have gone through a massive upgrade to handle minimum bias PbPb interactions at 50 kHz. This will results in an event rate 100 times higher than during Run 1 (2010-2012).
```